acs/DND.bcs

```diff
@@ -1146,6 +1143,10 @@ Script "DnD Remove Check Special" (int sp, int extra) CLIENTSIDE {
 		if(ConsolePlayerNumber() != (sp & 0xFFFF) - 1)
 			Thing_Remove(0);
 	}
+	else if(extra == DND_SPECIAL_TALENTCAPSULE) {
+		if(ConsolePlayerNumber() != (sp & 0xFFFF) - 1)
```

Comment:
You can just combine this with the above if statement using ||

Reply:
you're right.